utility/src/main/java/org/oppia/util/data/UrlImageParser.kt

```diff
@@ -0,0 +1,61 @@
+package org.oppia.util.data
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Rect
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.graphics.drawable.LevelListDrawable
+import android.text.Html
+import android.widget.TextView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.target.SimpleTarget
+import com.bumptech.glide.request.transition.Transition
+import java.net.URL
+
+/** UrlImage Parser for android TextView to load Html Image tag. */
+class UrlImageParser(internal var tvContents: TextView, internal var context: Context) : Html.ImageGetter {
+
+  var targets: ArrayList<BitmapTarget>? = null
+  /***
+   * Get the Drawable from URL
+   * @param urlString contains the source of the image
+   * @return urlDrawable
```

Comment:
Check this link and update accordingly:
https://github.com/oppia/oppia-android/blob/develop/data/src/test/java/org/oppia/data/backends/api/MockQuestionPlayerService.kt

Reply:
updated thanks.